keyboards/sofle_pico/readme.md

```diff
@@ -0,0 +1,43 @@
+# Sofle Pico Keyboard
+
+![Sofle Pico](https://raw.githubusercontent.com/JellyTitan/Sofle-Pico/main/docs/images/qmk/sofle_pico_hero_image.jpeg)
+![Sofle Pico PCB](https://raw.githubusercontent.com/JellyTitan/Sofle-Pico/main/docs/images/qmk/sofle_pico_pcb_hero_image.jpeg)
```

Comment:
These images need to be rehosted on Imgur.

Reply:
Done! (The legacy Sofle had them hosted on imgur - i cribbed em!)

Reply:
I'm terribly embarrassed - i thought i made this change months ago. I updated just now:
https://github.com/qmk/qmk_firmware/pull/23338/commits/d8fd17db1cfa1f0ac2417bc1afb8fae846f847c9